cli/azd/pkg/cloud/cloud.go

```diff
@@ -0,0 +1,94 @@
+package cloud
```

Comment:
We'll have to leave this up for voting, but I prefer if we can start sticking to a single `azure` package. I'm not sure if this causes cyclic dependencies currently because of how splayed the azure clients are currently

Reply:
Switching cloud.go to `azure` and updating references does not result in an import cycle. This is likely preferable given the relative simplicity of cloud.

Reply:
We can refactor this later. It's possible and fairly straightforward.